my_enumerable.rb

```diff
@@ -0,0 +1,36 @@
+# require './my_enumerable.rb'
```

Comment:
- [ ] Kindly remove this commented line of code, more so I don't think it's needful to import a file inside the same file. ℹ️